dataloader/load_extract.py

```diff
@@ -189,6 +189,7 @@ def _new_transfer_course(self, dft):
             course_code=dft[1],
             name=dft[2],
             credit_hours=dft[3],
+            course_type=get_course_type(dft[1]),
```

Comment:
good catch, thx. Can you use
`get_course_type(course_code)`
instead?

Reply:
I tried course_code but it says that it can't recognize course_code so I just used dft[1] instead